metricbeat/module/redis/info/info.go

```diff
@@ -95,6 +95,18 @@ The document sent to elasticsearch has the following structure:
 	      "total_connections_received": "146",
 	      "total_net_input_bytes": "2247",
 	      "total_net_output_bytes": "277354"
+	    },
```

Comment:
@chrsblck Yes, or put it in a different doc. The reason I put it in is that it shows up in the docs about the metricset and we do not include all fields, only almost all fields. +1 on finding a better way.